cluster/stone-prd-rh01/tenants/osde2e-cicada-tenant/component.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  name: osde2e-main
+  annotations:
+    appstudio.openshift.io/pac-provision: request
+    build.appstudio.openshift.io/request: configure-pac
+    build.appstudio.openshift.io/pipeline: '{"name":"docker-build","bundle":"latest"}'
+    image.redhat.com/generate: "true"
```

Comment:
I think this annotation is not needed anymore since you are using the ImageRepository. Could you point me to where you got this annotation so I can update it?

Reply:
I can remove it! I was following the previous efforts from @\gurnben  https://github.com/redhat-appstudio/tenants-config/blob/main/auto-generated/cluster/stone-prd-rh01/tenants/crt-redhat-acm-tenant/appstudio.redhat.com_v1alpha1_component_siteconfig-main.yaml#L7

Reply:
I can't update those :sweat_smile: 

yeah, please remove that one and the `appstudio.openshift.io/pac-provision: request` since that one is not being used already anymore as well

Reply:
updated!